internal/dataplane/translator/translator_test.go

```diff
@@ -4166,7 +4166,7 @@ func TestGetEndpoints(t *testing.T) {
 			if testCase.clusterDomain != "" {
 				clusterDomain = testCase.clusterDomain
 			}
-			result := getEndpoints(zapr.NewLogger(zap.NewNop()), testCase.svc, testCase.port, testCase.proto, testCase.fn, testCase.isServiceUpstream, clusterDomain)
+			result := getEndpoints(zapr.NewLogger(zap.NewNop()), testCase.svc, testCase.port, testCase.proto, testCase.fn, testCase.isServiceUpstream, clusterDomain, false)
```

Comment:
it would be great to add a set of unit test cases for the drain support in `TestGetEndpoints`. then everything else LGTM

Reply:
Added, thank you! @sandromodarelli